.github/workflows/check_code_snippets.yml

```
@@ -13,7 +13,7 @@ jobs:
       - name: Find Python scripts
         id: list-scripts
         run: |
-           SCRIPTS=$(find scripts -name "*.py" | jq -R -s -c 'split("\n")[:-1]')
+           SCRIPTS=$(find code_examples/source -name "*.py" | jq -R -s -c 'split("\n")[:-1]')
```

This is a bit greedy. At a minimum I'd add `-type f` so you don't find directories that end in `.py`. I'm also confused why you have to take something that is going to be multi-line input with one result per line, join it up, and then split it again right away?
```suggestion
           SCRIPTS=$(find 'code_examples/source' -type f -name "*.py" | jq -R -s -c 'split("\n")[:-1]')
```